ci/ssm.py

```diff
@@ -0,0 +1,253 @@
+                                  ' var_type must be \"String\" '
+                                  ' or \"SecureString\" \n'.format(key))
+                            print("Found value: {} \nExpected value:"
+                                  " \"String\" or \"SecureString\""
+                                  "\n".format(var['var_type']))
+                            exit()
+                except KeyError as e:
+                    print('A variable block in the {} section was found'
+                          ' without a var_type declaration'.format(key))
+                    exit()
+        print("No potential issues found.")
+    except KeyError as e:
+        print("variables.yaml is not formatted correctly.")
+        raise
+
+
+def put_parameter(name, value, var_type):
+    """Set parameters in ssm."""
+    # print("updating var: {}".format(name))
+    ssm = boto3.client('ssm')
+    ssm.put_parameter(
+        Name=name,
+        Description='',
+        Value=value,
+        Type=var_type,
+        Overwrite=True
+    )
+
+
+def parse_local_parameters(param_obj, verbose=None):
+    """Parse the source parameter object."""
+    for key in param_obj:
+        for var in param_obj[key]:
+            name = "/spel/{}/{}".format(key, var['var_name'])
+            value = var['value']
+            var_type = var['var_type']
+            if verbose:
+                print("Adding parameter:")
+                print("     Name: {}".format(name))
+                print("     Value: {}".format(value))
+                print("     Type: {}".format(var_type))
+                print("\n")
+            put_parameter(name, value, var_type)
+
+
+def get_parameters(client, path, **kwargs):
+    """Get SSM parameters."""
+    next_token = kwargs.get('next_token', None)
+    verbose = kwargs.get('verbose', None)
+    if next_token:
+        response = client.get_parameters_by_path(
+            Path=path,
+            Recursive=True,
+            WithDecryption=True,
+            MaxResults=10,
+            NextToken=next_token
+        )
+        if verbose:
+            print("Recieved NextToken with response: {}".format(response))
+    else:
+        response = client.get_parameters_by_path(
+            Path=path,
+            Recursive=True,
+            WithDecryption=True,
+            MaxResults=10
+        )
+        if verbose:
+            print("Recieved response: {}".format(response))
+    return response
+
+
+def get_env_paths():
+    """Get environment variable (ssm) paths."""
+    environment_paths = []
+    zones = [
+        "commercial",
+        "govcloud"
+    ]
+    project_types = [
+        "ci",
+        "rc",
+        "official"
+    ]
+    for type in project_types:
+        for zone in zones:
+            environment_paths.append("/spel/{}_{}".format(zone, type))
+    return environment_paths
+
+
+def get_ssm_path(env):
+    """Return the ssm path for an environment."""
+    return "/spel/{}/".format(env)
```

Comment:
This `/spel/` is hardcoded in a few places. It should be a parameter, or a global constant.

Reply:
moved all references to `/spel/` to global constant `SPEL_PATH`